.github/workflows/pipeline.yml

```diff
@@ -119,18 +120,30 @@ jobs:
       working-directory: ${{env.odbodsPath}}
 
     # Commit changes
-    - name: Git config
+    - name: Git config JKAN
       working-directory: ${{env.jkanPath}}
       run: |
-        git config user.name "GitHub Actions"
-        git config user.email "<>"
-    - name: Git status
+        git config user.name 'GitHub Actions'
+        git config user.email '<>'
+    - name: Git status JKAN
       run: git status
       working-directory: ${{env.jkanPath}}
-    - name: Commit and push
+    - name: Commit and push JKAN
       if: ${{ github.event.inputs.commitToSite == 'true' || github.event.schedule == '0 12 * * FRI' }} 
-      run: git add -A && git commit -m "Dataset Sync" && git push origin gh-pages
+      run: git add -A && git commit -m 'Dataset Sync' && git push origin gh-pages
       working-directory: ${{env.jkanPath}}
+    - name: Git config the_od_bods
+      working-directory: ${{env.odbodsPath}}
+      run: |
+        git config user.name "GitHub Actions"
+        git config user.email "<>"
+    - name: Git status the_od_bods
+      run: git status
+      working-directory: ${{env.odbodsPath}}
+    - name: Commit and push the_od_bods
+      if: ${{ github.event.inputs.commitToSite == 'true' || github.event.schedule == '0 12 * * FRI' }} 

```

Comment:
Minor nitpick but we should maybe change the `commitToSite` workflow parameter to `commitToRepos`. Or we could introduce a second parameter for `commitToODBods` if we wanted that level of granularity when debugging.

Reply:
changed to `commitToRepos`. I take your point about debugging - see how we go and if we need it later.